### DISCUSSION TITLE
What is the function of proposals in ava_dense_proposals_train.FAIR.recall_93.9.pkl during training slwofast?

### DISCUSSION THREAD

**Discussion by ZJU-lishuang** [open-mmlab/mmaction2#712] [Q&A] [answered]
Now I try to convert custom dataset to ava format.I got a csv like ava.
And I find the csv file already has the box coordinates.
What is the function of proposals in ava_dense_proposals_train.FAIR.recall_93.9.pkl during training slwofast?

**Comment by ZJU-lishuang** [comment, marked as answer]
Answer myself.:disappointed:
We can replace the proposals box with the gt box during the training slowfast.

**Reply by hotelbread** [reply]
how did you do that? And should I have to do that?
